docs/docs/reference/migration.md

```diff
@@ -3,51 +3,80 @@
 This document describes breaking changes and migrations between Constellation releases.
 Use [`constellation config migrate`](./cli.md#constellation-config-migrate) to automatically update an old config file to a new format.
 
+## Migrations to v2.19.1
+
+### Azure
+
+* During the upgrade, security rules are migrated and the old ones need to be cleaned up manually by the user. The below script shows how to delete them through the Azure CLI:
+
+```bash
+#!/bin/bash
+name="<insert>"  # the name provided in the config
+id="<insert>"    # the cluster id
+
+rules=(
+  "kubernetes"
+  "bootstrapper"
+  "verify"
+  "recovery"
+  "join"
+  "debugd"
+  "konnectivity"
+)
+
+for rule in "${rules[@]}"; do
+  echo "Deleting rule: $rule"
+  az network nsg rule delete \
+    --resource-group "$name-rg" \
```

Comment:
We need to also define the rg name as the other variables above since it can be specified in the constellation-conf.yaml und `.provider.azure.resourceGroup`